app/src/main/resources/database/scripts/initial-schema.sql

```diff
@@ -6,4 +6,12 @@ CREATE TABLE recipe (
 	target_vg smallint NOT NULL,
 	base_nicotine smallint NOT NULL,
 	quantity smallint
+);
+
+CREATE TABLE recipe_flavourings (
+    recipe_id serial UNIQUE NOT NULL,
+    flavouring_id int NOT NULL PRIMARY KEY,
```

Comment:
Move the **PRIMARY KEY** to the first line in the ```CREATE TABLE``` statement, this does not affect the functionality but it's better to organize the code in that way, first **PRIMARY KEYS**, second **FOREIGN KEYS** and then normal fields of the table.

Although on a second analysis, the ```flavouring_id``` should not be the **PRIMARY KEY** on this table because we will have a table of flavors as well. So in this table, both recipe_id and flavouring_id will reference other tables, and both fields together will be the primary key to this table, so we will have a composite **PRIMARY KEY**: check this link: https://stackoverflow.com/questions/5835978/how-to-properly-create-composite-primary-keys-mysql